shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -232,8 +232,11 @@ impl CassandraSinkCluster {
                 let nodes_shared = self.topology_task_nodes.read().await;
                 self.local_nodes = nodes_shared.clone();
             }
+            for node in &self.local_nodes {
+                tracing::error!("local nodes: {} {}", node.address, node.is_up);
```

Comment:
Wouldn't we only want to print this if it's down?

Reply:
Ooops those were from debugging, I'll remove